network/client.go

```diff
@@ -0,0 +1,215 @@
+package network
```

Comment:
i would call this package: `jira` this way creating a new jira client looks like: `jira.NewClient`